grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -1603,7 +1604,7 @@ class DocumentService {
                     documents {
```

Comment:
are we sure this is not nullable?

Reply:
I believe that in this situation INNER JOIN actually makes sense because we are querying **Order** table for documents and if documents does not exists on Order then we want to skip it. Do you disagree?